suites/multimds/basic/mount/fuse/default-permissions/yes.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      client:
+        fuse default permissions: true
```

Comment:
I believe it works although the process of figuring it out was silly. Take a look at these:

http://pulpito.ceph.com/pdonnell-2016-08-10_03:03:20-multimds-master---basic-mira/

Although I actually just found a bug in recovery where it does both:

http://pulpito.ceph.com/pdonnell-2016-08-11_20:40:52-multimds-master-testing-basic-mira/358642/

(So yes... tricky and maybe not worth it. In any case... this change was made by a temporary hack commit to see if `fuse_default_permissions` would resolve apparent hangs in the MDSs.)